File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void buildPyramid(int height);

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    buildPyramid(height);
}

void buildPyramid(int height)
{
    for (int i = 0; i < height; i++)
    {
        for (int j = 1; j <= height; j++)
        {
            if (j < height - i)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: week6/sentimental-mario-more/mario.py
import cs50

while True:
    height = cs50.get_int("Height: ")
    if height > 0 and height < 9:
        break

for i in range(height):
    print(f" " * (height - i - 1), end="")
    print(f"#" * (i + 1), end="")
    print(" " * 2, end="")
    print(f"#" * (i + 1), end="")
    print("")
